Marquee/Cells/Table Cell/TableViewCell.swift

```diff
@@ -20,7 +20,7 @@ class TableViewCell: UITableViewCell, UICollectionViewDelegate, UICollectionView
     
     var delegate: collectionViewDelegate?
     
-    let viewController = ViewController()
+    weak var viewController = ViewController()
```

Comment:
Maybe consider thinking of a more meaningful name for `ViewController()`. What kind of view controller is this? I can't tell by reading the name.